README.md

```diff
@@ -45,8 +45,20 @@ to automatically discover open remote debugging sessions and connect to them:
 
 This will open a Python process that listens for new debugger sessions and
 automatically connects to them for you.  If your breakpoint is run on
-an _entirely different host_, you can optionally specify the hostname where
-`sbd-listen` is running:
+an _entirely different host_ (e.g. `10.0.0.1`) you can optionally specify the
+hostname where `sbd-listen` is running:
```

Comment:
Typo `sdb`.

Reply:
I typo'ed the environment variables as well, unbelievable. You should publish an alias!